mmaction/models/backbones/c2d_r50.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Tuple, Union
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+
+from mmaction.models.backbones.resnet import ResNet
+from mmaction.registry import MODELS
+
+
+@MODELS.register_module()
+class C2D_R50(ResNet):
```

Comment:
It is not necessary to limit the structure to `R50`,  a `R101` counterpart can be built in the same way, so it's better to change `C2D_R50` to `C2D`.